src/test/java/hw2/TestEx1.java

```diff
@@ -0,0 +1,127 @@
+package hw2;
+
+import org.assertj.core.api.JUnitSoftAssertions;
+import org.junit.Rule;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestEx1 extends ChromeTest{
+
+    @Test
+    public void TestsEx1(){
+        homepage();
+        login();
+        header();
+        icons();
+        iconsText();
+        frameButton();
+        leftSection();
```

Comment:
Method names should bring value of what was checked inside. Try to specify it according to what you did in these methods.